docs/releases/1.11.txt

```diff
@@ -364,6 +364,10 @@ Miscellaneous
   :meth:`~django.db.models.fields.related.RelatedManager.set` now
   clear the ``prefetch_related()`` cache.
 
+* :func:`~django.test.utils.setup_test_environment` now raises an exception
+  if called a second time before calling
+  :func:`~django.test.utils.teardown_test_environment`.
+
```

Comment:
@claudep, @timgraham requested it here: https://github.com/django/django/pull/7032#issuecomment-237967038